Makefile

```diff
@@ -65,7 +65,11 @@ bump-major:
 	bumpversion major
 
 release:
-	python setup.py register sdist bdist_wheel upload --sign --identity="$(PGPIDENT)"
+    python -m pip install --upgrade build setuptools wheel twine
+	python setup.py register
```

Comment:
that direct invocation of setup.py should also be removed, no?

Reply:
Discussed at https://github.com/celery/django-celery-beat/pull/737#issuecomment-1966276680

Reply:
Ah, I did not make the link, sorry.